test/DualTest.jl

```diff
@@ -7,6 +7,11 @@ using ForwardDiff: Partials, Dual, value, partials
 import NaNMath
 import Calculus
 
+if VERSION >= v"0.6.0-dev.2767"
+    Pkg.installed("SpecialFunctions") === nothing && Pkg.add("SpecialFunctions")
```

Comment:
not correct when installed elsewhere on LOAD_PATH, and does not correctly remove the package after testing if it wasn't otherwise asked for